src/components/ui/Footer.tsx

```diff
@@ -121,62 +162,31 @@ const Footer = () => {
               </Text>
               <Flex gap="2">
                 {SOCIAL_LINKS.map((social) => (
-                  <a href={social.link} key={social.name} target="_blank">
-                    <Image
-                      src={`/images/social-icons/${social.name}.svg`}
-                      alt={social.name}
-                      width={40}
-                      height={40}
-                    />
+                  <a
+                    href={social.link}
+                    key={social.name}
+                    target="_blank"
+                    className="hover:!bg-navy-500 hover:!text-white"
+                    style={{
+                      ...buttonStyle,
+                      borderRadius: "100%",
+                      padding: "var(--space-1)",
+                    }}
+                  >
+                    <social.icon size={20} />
                   </a>
                 ))}
               </Flex>
-
-              <Box
-                asChild
-                mt="4"
-                style={{
-                  cursor: "pointer",
-                  backgroundColor: "white",
-                  color: "var(--accent-11)",
-                  borderRadius: "var(--radius-3)",
-                  padding: "var(--space-3) var(--space-6)",
-                  width: "max-content",
-                  textTransform: "capitalize",
-                  fontWeight: 500,
-                  transition: "all 200ms",
-                }}
-                display={{ initial: "none", sm: "block" }}
-                className="hover:!bg-navy-500 hover:!text-white"
-              >
-                <Link href="/donate" target="_blank" rel="noreferrer">
-                  Donate
```

Comment:
the HRTKits page (on `saga`) uses a similar button that I think could be reused here?